tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -676,8 +676,9 @@ public:
    /// hx["nominal"].Draw();
    /// hx["pt:down"].Draw("SAME");
    /// ~~~
-   /// The run-time of Vary does not scale linearly with the number of variations. This behaviour stems from Vary
-   /// booking the variations within a single event loop instead of multiple.
+   /// The run-time of Vary does not scale linearly with the number of variations. 
```

Comment:
I probably missed the original discussion that led to this, but why do you feel that this extra comment is needed? Was this unexpected for you?

Reply:
Yes, it was, thus, I thought I was doing something wrong because of that, but I wasn't. So Axel and I thought it'd be a good idea to document that, so he told me to make a PR.